### PR TITLE
Add README and travis configs from master

### PR DESCRIPTION
**Pull request by mariuz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+branches:
+  only:
+    - master
+    - B3_0_Release
+
+install:
+  # wget --no-check-certificate https://www.cmake.org/files/v3.2/cmake-3.2.3-Linux-x86_64.sh
+  # sudo sh cmake-3.2.3-Linux-x86_64.sh --skip-license --prefix=/usr
+  - sudo apt-get update
+  - sudo apt-get install -y libtommath0 libtommath-dev libtomcrypt0 libtomcrypt-dev
```
Comment (asfernandes) [inline .travis.yml]: This libraries are not necessary for v2.5.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  # sudo python -m pip install Mako fdb
+  # svn co http://svn.code.sf.net/p/firebird/code/qa/fbtest/trunk/ fbtest
+  # svn co http://svn.code.sf.net/p/firebird/code/qa/fbt-repository/trunk/ fbtest/fbt
+
+script:
+  - ./autogen.sh --enable-binreloc --prefix=/opt/firebird
+  - make -j4
```
Comment (asfernandes) [inline .travis.yml]: v2.5 cannot use `make -j` AFAIR. It should use `make CPU=N`.

**Review by asfernandes — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (mariuz) [reply .travis.yml]: Thank you , i have removed the libs

File: .travis.yml (hunk 2 above)
Comment (mariuz) [reply .travis.yml]: Thank you i have removed the -j flag

**Review by asfernandes — APPROVED** [review]

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+notifications:
+  email: false
+
+sudo: required
+dist: trusty
+
+branches:
+  only:
+    - master
+    - B3_0_Release
```
Comment (asfernandes) [inline .travis.yml]: @mariuz B2_5_Release is not listed here.

Did you tested the builds commands in Ubuntu trusty? I do not remember about this specific version, but I always have trouble building 2.5 in newer Ubuntu versions.

**Review by asfernandes — CHANGES_REQUESTED** [review]

**Comment by asfernandes** [comment]
@mariuz now that B2_5_Release is added, there is already a travis log in this pull request, with errors.

**Comment by mariuz** [comment]
I have added the sudo for make dist , seems that is needed for 2.5 branch

File: .travis.yml (hunk 3 above)
Comment (mariuz) [reply .travis.yml]: On trusty firebird 2.5 branch builds ok 

On 17.10 i have an error (segfault when create_db empty.fdb)

File: builds/install/arch-specific/linux/Makefile.in [hunk 4]
```diff
@@ -76,20 +76,17 @@ PackageDir:=$(shell if [ -d $(RedhatPackageDir) ]; then echo $(RedhatPackageDir)
...
-#Target for required libraries detection
-RpmRequiredVersionTarget=$(GEN_ROOT)/firebird/bin/isql
 #Curses library
 TermLib=@TERMLIB@.so
 
 
 dist : packages
 
-packages: tarfile rpmfile debugfile
+packages: tarfile debugfile
```
Comment (asfernandes) [inline builds/install/arch-specific/linux/Makefile.in]: Why are you removing things deprecated later from 2.5 branch?

Comment (mariuz) [reply builds/install/arch-specific/linux/Makefile.in]: I did a diff on this file with 3.0 branch and merged the removal of rpmfile building (related to http://tracker.firebirdsql.org/browse/CORE-5180) , i have asked also on devel list if it is ok

**Pull request closed without merging**